addons/mail/static/src/components/welcome_view/welcome_view.js

```diff
@@ -11,21 +11,28 @@ export class WelcomeView extends owl.Component {
      */
     setup() {
         super.setup();
-        useRefToModel({ fieldName: 'guestNameInputRef', modelName: 'mail.welcome_view', propNameAsRecordLocalId: 'localId', refName: 'guestNameInput' });
-        useUpdateToModel({ methodName: 'onComponentUpdate', modelName: 'mail.welcome_view', propNameAsRecordLocalId: 'localId' });
+        useRefToModel({ fieldName: 'guestNameInputRef', modelName: 'mail.welcome_view', propNameAsRecordLocalId: 'welcomeViewLocalId', refName: 'guestNameInput' });
+        useUpdateToModel({ methodName: 'onComponentUpdate', modelName: 'mail.welcome_view', propNameAsRecordLocalId: 'welcomeViewLocalId' });
+    }
+
+    mounted() {
+        if (this.welcomeView.channel.defaultDisplayMode === 'video_full_screen') {
+            this.welcomeView.mediaPreview.enableMicrophone();
+            this.welcomeView.mediaPreview.enableVideo();
+        }
```

Comment:
This should not be in component. Call the methods in the boot file, and use "do..." variables if it must be done specifically after render for some reason (but I don't think that should be the case).